src/main/kotlin/racingcar/ResultView.kt

```diff
@@ -10,9 +10,19 @@ class ResultView(private val progressString: String = "-") {
     }
 
     private fun displayCar(car: Car) {
-        repeat(car.now()) {
-            print(progressString)
+        print("${car.name} : ")
+        repeat(car.progress) {
+            print(progressCharacter)
         }
         println()
     }
+
+    fun displayWinner(cars: List<Car>) {
+        val names = cars.map { car ->
+            car.name
+        }
+
+        val joinName = names.joinToString()
+        println("${joinName}가 최종 우승했습니다. ")
```

Comment:
```suggestion
        println("${names.joinToString()}가 최종 우승했습니다. ")
```
변수를 활용하지 않는다면, 변수를 별도로 정의하지 않고 인라인으로 작성해도 좋을 것 같아요.